src/runtime/virtcontainers/utils/utils.go

```diff
@@ -399,12 +399,14 @@ func MkdirAllWithInheritedOwner(path string, perm os.FileMode) error {
 		info, err := os.Stat(curPath)
 
 		if err != nil {
-			if err = os.Mkdir(curPath, perm); err != nil {
+			err = os.Mkdir(curPath, perm)
```

Comment:
What about just calling `os.MkdirAll` and do not care about EEXIST errors?

Reply:
Good idea, changed the code to use MkdirAll.